iceoryx2/src/prelude.rs

```diff
@@ -14,7 +14,7 @@ pub use crate::iox2::Iox2;
 pub use crate::iox2::Iox2Event;
 pub use crate::port::event_id::EventId;
 pub use crate::service::{
-    attribute::AttributeSet, attribute::AttributeSpecifier, attribute::RequiredAttributes,
+    attribute::AttributeSet, attribute::AttributeSpecifier, attribute::AttributeVerifier,
```

Comment:
```suggestion
    attribute::{AttributeSet, AttributeSpecifier, AttributeVerifier},
```
Not required to change now but it would save you some typing for the next time and my old eyes wouldn't have to travel that much ;)

Reply:
Oh, I never type this, this is all the rust-analyzer.